app/widgets/common/deployModal/EnvironmentDropdown/EnvironmentDropdown.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { Form } from 'cloudify-ui-components';
+import type { DropdownProps } from 'semantic-ui-react';
+import type { DynamicDropdownProps } from '../../components/DynamicDropdown';
+import { useBoolean } from '../../../../utils/hooks';
+import { filterEnvironments, mapFetchedEnvironments, useFetchTrigger } from './EnvironmentDropdown.utils';
+import SearchActions from '../../actions/SearchActions';
+import { FilterRuleOperators, FilterRuleType } from '../../filters/types';
+import type { BlueprintRequirements } from '../../blueprints/BlueprintActions';
+import type { Environment, FilteredEnvironments } from './EnvironmentDropdown.types';
+import EnvironmentDropdownList from './EnvironmentDropdownList';
+import type { EnvironmentDropdownListProps } from './EnvironmentDropdownList';
+import { defaultEnvironmentList } from './EnvironmentDropdown.consts';
+
+interface EnvironmentDropdownProps {
+    value?: string;
+    name: DynamicDropdownProps['name'];
+    placeholder: DynamicDropdownProps['placeholder'];
```

Comment:
Probably a leftover from the original implementation. I guess we should have here:

```suggestion
    name: DropdownProps['name'];
    placeholder: DropdownProps['placeholder'];
```

Am I correct?

Reply:
Yeah, you're 100% right
Fixing it right away 💪

Reply:
Applied the fix as a part of 352bef7f761e8cc9fa840318d740cd91bc589299